system_files/desktop/shared/usr/libexec/bazzite-user-setup

```diff
@@ -33,7 +33,7 @@ USER_SETUP_FEDORA_VER_RAN=$(cat $USER_SETUP_FEDORA_VER_FILE)
 USER_SETUP_IMAGE_VER_RAN=$(cat $USER_SETUP_IMAGE_VER_FILE)
 
 # Run script if updated
-if [[ -f $USER_SETUP_VER_FILE && -f $USER_SETUP_FEDORA_VER_RAN && -f $USER_SETUP_FEDORA_VER_FILE ]]; then
+if [[ -f $USER_SETUP_VER_FILE && -f $USER_SETUP_FEDORA_VER_FILE && -f $USER_SETUP_FEDORA_VER_FILE ]]; then
```

Comment:
Looks like you doubled up on the `USER_SETUP_FEDORA_VER_FILE` check. Maybe should be the below against the three files or just drop one of the checks if not needed? 

Also perhaps move the below three `cat` lines into the first if after the file exist checks?

```bash
USER_SETUP_VER_RAN=$(cat $USER_SETUP_VER_FILE)
USER_SETUP_FEDORA_VER_RAN=$(cat $USER_SETUP_FEDORA_VER_FILE)
USER_SETUP_IMAGE_VER_RAN=$(cat $USER_SETUP_IMAGE_VER_FILE)
```

```suggestion
if [[ -f $USER_SETUP_VER_FILE && -f $USER_SETUP_FEDORA_VER_FILE && -f $USER_SETUP_IMAGE_VER_FILE ]]; then
```

Reply:
Good catch! Thank you! Since this already got merged, you want to PR just the one line change?

I'll approve.

Reply:
No problem! #1023